### DISCUSSION TITLE
Open HTML table and R help in right tab (using R)

### DISCUSSION THREAD

**Discussion by pehkawn** [microsoft/vscode-jupyter#10144] [Questions and Answers]
I've noticed that when I try run a function that will generate a html table (e.g. `sjPlot::view_df()` ) , in the Jupyter extension, the table will open in my browser. Also, if I try access R Help, Jupyter will generate an erroneous ouput inside the notebook.

Instad, I was wondering if it is possible to configure Jupyter so that HTML tables and R Help in a separate tab on the right hand side in VS Code?

**Comment by rchiodo** [comment]
Can you give repro steps? I'm not sure exactly what's happening when you generate the table.

**Comment by pehkawn** [comment]
Ok, here's two examples:

Generating a HTML table using `sjPlot::view_df()`:
```R
df <- data.frame(
    A = structure(ordered(1:3, labels = letters[1:3]), label = "variable A"),
    B = structure(ordered(4:6, labels = letters[4:6]), label = "variable B")
)

sjPlot::view_df(df)
```
Output opens in my browser:
![image](https://user-images.githubusercontent.com/44672121/171390497-15a03935-7d03-4488-ac4d-fd37b07183d3.png)
As you can see, the generated table opens in my browser. I'd rather it opened in a right side tab in VS Code.

Calling R Help from Jupyter Notebook:
```R
?sjPlot::view_df()
```
Output below cell in Jupyter notebook:
![image](https://user-images.githubusercontent.com/44672121/171391442-ff818763-14ff-462a-8d72-3006e560f96d.png)
As you can see, the formatting is all screwed up.
### Desired output:
When calling R Help from R terminal,
```R
r$> sjPlot::view_df()
```
R Help now opens in a right side tab:
![image](https://user-images.githubusercontent.com/44672121/171392257-ea195a45-23f7-4eeb-97a9-8571392397bc.png)
